test/helper.ts

```diff
@@ -0,0 +1,17 @@
+/*
+
+ Screeps Typescript Test Helper
+
+ We add the following to the global namespace to mimic the Screeps runtime:
+ + lodash
+ + Screeps game constants
+
+ */
+//
+
+const _ = require("lodash");
```

Comment:
Might need to declare `global` to prevent any IDE errors, so do the following for this file:

```ts
declare var global: any;

import * as _ from "lodash";
import consts from "./mock/game";

global._ = _;
_.merge(global, consts);
```